pkg/api/handlers/compat/images.go

```diff
@@ -353,22 +354,20 @@ func imageDataToImageInspect(ctx context.Context, l *libimage.Image) (*handlers.
 	if err != nil {
 		return nil, err
 	}
-	ports, err := portsToPortSet(info.Config.ExposedPorts)
-	if err != nil {
-		return nil, err
-	}
 
 	// TODO: many fields in Config still need wiring
-	config := dockerContainer.Config{
-		User:         info.User,
-		ExposedPorts: ports,
-		Env:          info.Config.Env,
-		Cmd:          info.Config.Cmd,
-		Volumes:      info.Config.Volumes,
-		WorkingDir:   info.Config.WorkingDir,
-		Entrypoint:   info.Config.Entrypoint,
-		Labels:       info.Labels,
-		StopSignal:   info.Config.StopSignal,
+	config := dockerSpec.DockerOCIImageConfig{
+		ImageConfig: imageSpec.ImageConfig{
+			User:         info.User,
+			ExposedPorts: info.Config.ExposedPorts,
+			Env:          info.Config.Env,
+			Cmd:          info.Config.Cmd,
+			Volumes:      info.Config.Volumes,
+			WorkingDir:   info.Config.WorkingDir,
+			Entrypoint:   info.Config.Entrypoint,
+			Labels:       info.Labels,
+			StopSignal:   info.Config.StopSignal,
+		},
```

Comment:
`info` can include healthcheck infornation, and `config` can, too.  Should it be added here?

Reply:
Fair enough.